### DISCUSSION TITLE
A way for themes?

### DISCUSSION THREAD

**Discussion by pyroboots** [nesbox/TIC-80#2255] [Ideas]
I know this would probably be too complex as I start this discussion but I'll break it down. I think it would be a good idea to add themes: not colour changing and un-retro-ing the app but just icons like changing sprites and tiles in the `config.tic` file. This is already possible via the `import` command and just extracting the sprites from the compiled cart and applying the sheet in the `config.tic` file by copy and pasting the data. It would be nice to have it as a vanilla feature that you can choose public themes from the menu in the `surf` command.

Thanks for reading! 👍

**Comment by Skeptim** [comment]
I never realized this was possible, I like the idea !
However, I am not sure it's worth making it a vanilla feature. We could already add something about it on the wiki were people could put their files maybe.

**Comment by Skeptim** [comment]
I have created a [Themes](https://github.com/nesbox/TIC-80/wiki/Themes) page on the wiki ^^
